### DISCUSSION TITLE
Cannot use metamask in walletconnect v2 modal?

### DISCUSSION THREAD

**Discussion by zouhangwithsweet** [wevm/wagmi#2166] [Q&A] [answered]
I had installed the metamask extension in my browser. But in docs/example, there is no Metamask icon to Click.
<img width="831" alt="image" src="https://user-images.githubusercontent.com/21095710/231337258-96438e45-198e-41ac-9c61-8dee5a8a98e7.png">

**Comment by jxom** [comment, marked as answer]
I don't think MetaMask supports WalletConnect v2 yet.

**Reply by zouhangwithsweet** [reply]
I notice that there are some diffrent with https://web3modal.com/
<img width="368" alt="image" src="https://user-images.githubusercontent.com/21095710/231339965-ef12c427-edd3-4607-9d57-99cd6ea28c1a.png">

**Reply by jxom** [reply]
I believe that page is using WalletConnect v1 with Web3Modal.
